cmd/ef_tests/state/utils.rs

```diff
@@ -9,22 +9,25 @@ use ethrex_levm::db::{gen_db::GeneralizedDatabase, CacheDB};
 use ethrex_storage::{EngineType, Store};
 use ethrex_vm::{
     backends::revm::db::{evm_state, EvmState},
-    StoreWrapper,
+    DynVmDatabase, StoreVmDatabase,
 };
 
 /// Loads initial state, used for REVM as it contains EvmState.
-pub async fn load_initial_state(test: &EFTest) -> (EvmState, H256) {
+pub async fn load_initial_state(test: &EFTest) -> (EvmState, H256, Store) {
     let genesis = Genesis::from(test);
 
     let storage = Store::new("./temp", EngineType::InMemory).expect("Failed to create Store");
     storage.add_initial_state(genesis.clone()).await.unwrap();
 
+    let vm_db = DynVmDatabase(Box::new(StoreVmDatabase::new(
```

Comment:
Ideally, `DynVmDatabase` is an internal construct of vm and not exposed to the outside, but this module is leaking all over the place. Need to tackle this in a follow up.